migrations/0011_referrals_table.up.sql

```diff
@@ -0,0 +1,8 @@
+CREATE TABLE referrals
+(
+    account_name VARCHAR(13) REFERENCES users(account_name),
+    referral_id VARCHAR NOT NULL
+);
+
+CREATE INDEX referral_id_idx ON referrals (referral_id);
+
```

Comment:
should we create index on account_name as well to optimize getting a referral for a user?

Reply:
should we? primary key is unique by default

Reply:
yes, we should, it will be primary key because it's one-to-one relation